models/PyCryptoBot.py

```diff
@@ -42,6 +42,7 @@
 parser.add_argument('--verbose', type=int, help='verbose output=1, minimal output=0')
 parser.add_argument('--config', type=str, help="Use the config file at the given location. e.g 'myconfig.json'")
 parser.add_argument('--logfile', type=str, help="Use the log file at the given location. e.g 'mymarket.log'")
+parser.add_argument('--logDateFormat', type=str, help="Date format used in log. e.g '%m/%d/%Y %I:%M:%S %p'")
```

Comment:
General comment and personal though.
I'm not convince its a good idea to make it configurable; It makes the code a bit more complex and does not bring much value for user.

Reply:
For European people like it makes a lot of sense ;-)
But yeah I can remove it if you want.